curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-javascript-by-building-a-role-playing-game/62a24128d5e8af1b47ad1aab.md

```diff
@@ -0,0 +1,101 @@
+---
+id: 62a24128d5e8af1b47ad1aab
+title: Step 11
+challengeType: 0
+dashedName: step-11
+---
+
+# --description--
+
+Give both your `#controls` and `#stats` elements a `border` of `1px solid black`, a `black` text color, and `5px` of padding.
+
+# --hints--
+
+You should have a `#controls, #stats` selector.
+
+```js
+const selector = new __helpers.CSSHelp(document).getStyle('#controls, #stats');
```

Comment:
I do not want to have too many people reviewing one PR. But:

Would it help if the helper function had a flag to automatically search for permutations of selectors like this?

Reply:
> I do not want to have too many people reviewing one PR. But:

This is the first JS project so it's kind of important - no such thing as too many reviewers.

Yes, that might be a helpful feature, but low priority. We are only having campers write the HTML/CSS for the first couple of projects (to link their previous learning to these new concepts) before we just start providing it all for them and focus entirely on the JS.